models/thought.js

```diff
@@ -0,0 +1,9 @@
+import mongoose from 'mongoose'
+
+const Thought = mongoose.model('Thought', {
+  message: { type: String, requiered: true, minlength: 5, maxlength: 140 },
+  hearts: { type: Number, requiered: true, default: 0},
+  createdAt: { type: Date, default: Date.now }
```

Comment:
It will be nicer to read if you make every single attribute to take separate line :)